packages/animations/example/android/build.gradle

```diff
@@ -1,35 +1,16 @@
-buildscript {
-    ext.kotlin_version = '1.9.0'
-    repositories {
-        google()
-        mavenCentral()
-    }
-
-    dependencies {
-        classpath 'com.android.tools.build:gradle:8.5.1'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-    }
-}
-
 allprojects {
     repositories {
-        // See https://github.com/flutter/flutter/blob/master/docs/ecosystem/Plugins-and-Packages-repository-structure.md#gradle-structure for more info.
-        def artifactRepoKey = 'ARTIFACT_HUB_REPOSITORY'
-        if (System.getenv().containsKey(artifactRepoKey)) {
-            println "Using artifact hub"
-            maven { url System.getenv(artifactRepoKey) }
-        }
```

Comment:
You'll need to add the new artifact hub code; see https://github.com/flutter/packages/pull/7822/files for an example.